spec/openapi.yaml

```diff
@@ -1697,6 +1697,51 @@ paths:
             application/json:
               schema:
                 $ref: '#/components/schemas/404response'
+  /v1/messaging/numbers/sender_address/addresses/{id}/reverify:
+    post:
+      security:
+        - basic_auth: [ ]
+        - hmac_auth: [ ]
+      tags:
+        - Source Address
+      summary: Re-verify Sender Address
```

Comment:
We should have description for the API, please check others for do the same?
Thank you!

Reply:
Updated